src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqExpectationsRepository.kt

```diff
@@ -87,6 +84,28 @@ class JooqExpectationsRepository(dsl: DSLContext)
                 }
     }
 
+    override fun getAllExpectations(): List<TouchstoneModelExpectations>
+    {
+        val records = dsl.select(
+                SCENARIO_DESCRIPTION.DISEASE,
+                TOUCHSTONE.ID,
+                RESPONSIBILITY_SET.MODELLING_GROUP,
+                *BURDEN_ESTIMATE_EXPECTATION.fields())
```

Comment:
So this is missing outcomes and countries. The latter aren't actually needed for this front-end work, since we're looking instead at max applicable countries, but outcomes definitely are.

Reply:
There's a subsequent call in the map to withCountriesAndOutcomes, which looks like it should get both (though I should have added a test that it actually does!)
But if we don't need countries, I'll change that to just get outcomes (and add a test)